### PR TITLE
2891: Override meta description

### PR DESCRIPTION
**Pull request by f1sh1918**
### Short description

Override meta description for search engine purposes

### Proposed changes

- override meta description
- import overrides in buildConfig

### Side effects

- none

### Resolved issues

Fixes: #2891

---

### CODE REVIEW

File: translations/override-translations/obdach.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "categories": {
+    "de": {
+      "metaDescription": "{{appName}} - die mehrsprachige Plattform, die wohnungslose und von Wohnungslosigkeit bedrohte Menschen mit wichtigen Anlaufstellen wie Beratungen und Notunterkünften verbindet."
+    }
+  }
+}
```
Comment (LeandraH) [inline translations/override-translations/obdach.json:7]: Man, that description sounds like a lawyer wrote it but I guess that's not up to us to decide...

**Review by LeandraH — APPROVED** [review]

File: translations/override-translations/obdach.json (hunk 1 above)
Comment (f1sh1918) [reply translations/override-translations/obdach.json:7]: ugh, don't mess with @FritjofK :D

Comment (LeandraH) [reply translations/override-translations/obdach.json:7]: Good point, I don't really have a better suggestion either :D

**Review by steffenkleinle — APPROVED** [review]
Changed the metadescription for landing too.
@f1sh1918 @FritjofK keep in mind that this is probably already handled or overwritten by the CMS.

**Pull request merged**